hpx/lcos/local/barrier.hpp

```diff
@@ -46,9 +46,10 @@ namespace hpx { namespace lcos { namespace local
         /// releasing all waiting threads as soon as the last \a thread
         /// entered this function.
         void wait();
+        void increase();
```

Comment:
Let's align the name of this function with previous art. The closest would probably `std::latch::count_down()`. I'd suggest to rename this to `barrier::count_up()`. Would that work for everybody?